website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory-native.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Setting up SSO with Azure Active Directory"
+id: "setting-up-enterprise-sso-with-azure-active-directory-native"
+---
+
+
+:::info Enterprise Feature
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re
+interested in learning more about an Enterprise plan, contact us at
+sales@getdbt.com.
+:::
+
+dbt Cloud Enterprise supports single-sign on via Azure Active Directory (Azure AD).
+You will need permissions to create and manage a new Azure AD application.
+Currently supported features include:
+
+* IdP-initiated SSO
+* SP-initiated SSO
+* Just-in-time provisioning
+
+## Configuration
+
+dbt Cloud supports both single tenant and multitenant Azure Active Directory SSO
+Connections. For most Enterprise purposes, you will want to use the Single
+tenant flow when creating an Azure AD Application.
+
+### Creating an application
+
+Log into the Azure portal for your organization. Using the **Azure Active Directory** page, you will
+need to select the appropriate directory and then register a new application. 
+
+1. Under **Manage**, select **App registrations**
+2. Click **+ New Registration** to begin creating a new application
+3. Create the new app registration with the following configurations:
+
+| Field | Value |
+| ----- | ----- |
+| **Name** | dbt Cloud |
+| **Supported account types** | Accounts in this organizational directory only _(single tenant)_ |
+| **Platform configuration** | Client Application |
+
+4. Save the App registration to continue setting up Azure AD SSO
+
+
+<Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-app-registration-empty.png" title="Creating a new app registration"/>
+<Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-new-application.png" title="Configuring a new app registration"/>
+
+### Configuring the redirect URI
+
+:::note Redirect URIs
+If you are deploying dbt Cloud into a VPC, you should use the hostname where
+the dbt Cloud application is deployed instead of `https://cloud.getdbt.com` in
+the **Redirect URI** configuration shown below.
+:::
+
+
+5. Under **Manage**, click the **Authentication** link
+6. Add a **Platform configuration** for the dbt Cloud **Redirect URI**
+
+| Field | Value |
+| ----- | ----- |
+| Redirect URI | `https://cloud.getdbt.com/complete/azure` |
```

Comment:
1. For single tenants: ~https://cloud.getdbt.com/complete/azure~ https://cloud.getdbt.com/complete/azure_single_tenant
2. I dont see docs for multi tenants here but the uri would be: `https://cloud.getdbt.com/complete/azure_multi_tenant`

Reply:
updated! Thanks for the heads up :)